src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2.java

```diff
@@ -28,25 +28,25 @@
  *     The caller combines the DREAM challenge-winning somatic genotyping engine of the original MuTect
  *     (<a href='http://www.nature.com/nbt/journal/v31/n3/full/nbt.2514.html'>Cibulskis et al., 2013</a>) with the
  *     assembly-based machinery of <a href="https://www.broadinstitute.org/gatk/documentation/tooldocs/org_broadinstitute_gatk_tools_walkers_haplotypecaller_HaplotypeCaller.php">HaplotypeCaller</a>.
- *     Although we present the tool for somatic analyses, it may also apply to other contexts.
+ *     Although we present the tool for somatic analyses, it may also apply to other contexts, such as mitochondrial variant calling.
```

Comment:
Excited to see a workflow for mito.

Reply:
We have a req open for someone to develop a mitochondrial pipeline, but it's proving hard to fill.  :(

Reply:
Perhaps it could be a mini-project for someone internal?

Reply:
It should be noted that one can currently run M2 on mitochondria and get meaningful results, although one could undoubtedly improve upon off-the-shelf results with wome tweaking.